apps/signal/56-forms-and-signal/src/app/order.component.ts

```diff
@@ -59,13 +60,24 @@ export default class OrderComponent {
   });
 
   productId = input('1');
+  quantity = input(1);
```

Comment:
you could use `this.route.snapshot.queryParams['quantity']` since this.route is created first the value is available when you want to instantiate the class,
The code is simpler. but your code works. Waiting for signal forms to make all of this easy 🔥